fundingmanager.go

```diff
@@ -2846,6 +2847,11 @@ func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
 		minHtlc = f.cfg.DefaultRoutingPolicy.MinHTLC
 	}
 
+	chanReserve := f.cfg.RequiredRemoteChanReserve(capacity, ourDustLimit)
```

Comment:
if i'm reading this correctly, we'll only use the user chosen reserve if it is at least the our dust limit. it doesn't seem obvious that it would be ignored if the value is too low. should we check this sooner in the pipeline and return an error?

Reply:
I agree. I have shifted the input validation code to rpcserver.go similar to the way other rpc fields are being validated: https://github.com/lightningnetwork/lnd/pull/2708/files#diff-01ada5680aaa18fc79cbf78b2d55a37bR1366